commit ba25fda3aa561ae51bb90951b339379ef94f6197
Author: Mike Hiretsky <mh@calculate.ru>
Date:   Mon Jul 18 17:55:06 2011 +0400

    Fix smb.conf for work remote/distfiles in net-fs/samba-3.5.8.

diff --git a/profile/samba/etc/samba/smb.conf b/profile/samba/etc/samba/smb.conf
index abfde3a..5d8911d 100644
--- a/profile/samba/etc/samba/smb.conf
+++ b/profile/samba/etc/samba/smb.conf
@@ -160,6 +160,12 @@ chown=root:root
 	browseable = No
 	writable = yes
 	hide unreadable = yes
+#?pkg(net-fs/samba)>=3.5.8#
+	security mask = 0664
+	force create mode = 0664
+	force security mode = 0664
+	vfs objects = acl_xattr fake_perms
+#pkg#
 
 #Удаляем ресурс для FTP - сервис FTP не установлен
 #?sr_ftp_set==off#
@@ -175,4 +181,4 @@ chown=root:root
 	browseable = yes
 	writable = yes
 	hide unreadable = yes
-#sr_ftp_set#
\ No newline at end of file
+#sr_ftp_set#
